### ISSUE TITLE
[BUG] Microsoft.Playwrite.MSTest PageTest class looks for latest chromium browser when target browser = chrome

### ISSUE THREAD

**Issue by cwalker-reveille** [microsoft/playwright-dotnet#2445]
**Context:**
- Playwright Version: Microsoft.Playwright.MSTest 1.29.0
- Operating System: Windows
- .NET version: 4.8
- Browser: chrome
- Extra: 

**Code Snippet**

```cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Threading.Tasks;
using Microsoft.Playwright.MSTest;

namespace Reveille.ReveillePlaywright.Tests
{
    [TestClass()]
    public class TestRunnerTests : PageTest
    {
         [TestMethod()]
            public void DefaultsOnlyTest()
            {
                TestRunner t = new TestRunner( new Configuration { } );
                Assert.AreEqual( Configuration.DefaultBrowser, t.instanceConfig.Browser, "Browser Default" );
                Assert.AreEqual( Configuration.DefaultDebug, t.instanceConfig.Debug, "Debug Default" );
            }
     }
}
```
.runsettings
```xml
<?xml version="1.0" encoding="utf-8" ?>
<RunSettings>
	<Playwright>
		<BrowserName>chrome</BrowserName>
	</Playwright>
</RunSettings>
```

The tests are run targeting chrome (not chromium). When I upgraded Microsoft.Playwrite.MSTest from 1.28.0 to 1.29.0 and ran the tests,   I received this messge
```
 Initialization method Reveille.ReveillePlaywright.Tests.Page_GoTo_Tests.BrowserSetup threw exception. Microsoft.Playwright.PlaywrightException: Executable doesn't exist at C:\Users\Chris\AppData\Local\ms-playwright\chromium-1041\chrome-win\chrome.exe
╔════════════════════════════════════════════════════════════╗
║ Looks like Playwright was just installed or updated.       ║
║ Please run the following command to download new browsers: ║
║                                                            ║
║     pwsh bin/Debug/netX/playwright.ps1 install             ║
║                                                            ║
║ <3 Playwright Team                                         ║
╚════════════════════════════════════════════════════════════╝.
```
I am aware that each new release of Playwright targets the latest browser, in this case, chromium-1041. However I am only interested  the machine's version of Google Chrome (stock browser) per the documentation  <https://playwright.dev/dotnet/docs/browsers>.

I would expect the .runsettings file's browsername = chrome to not look for  
the chromium-1041 chrome but the Google Chrome.exe.

**Comment by mxschmitt** [comment]
For that you need to set the BrowserName to chromium and the LaunchOptions>Channel to "chrome": https://playwright.dev/dotnet/docs/test-runners#customizing-browserlaunch-options

**Comment by cwalker-reveille** [comment]
@mxschmitt 
Thanks for the VERY fast response. I immediately made the change in .runsettings but unfortunately got the same results. It was like it was ignoring the .runsettings values. I then remembered reading this issue #2353 and wondered if this was related.
I created 2 simple MSTest projects one using .Net 6 and the other using .Net 4.8. Sure enough the project using .Net 6 works as expected and the one using 4.8 does not. I uploaded these 2 projects if you want to try it. Thanks  

https://github.com/cwalker-reveille/ReveillePWNet.Test

https://github.com/cwalker-reveille/ReveillePWCore.Test
